skymp5-front/src/index.js

```diff
@@ -33,3 +33,5 @@ window.scrollToLastMessage = () => {
   const _list = document.querySelector('#chat > .list');
   if (_list != null) { _list.scrollTop = _list.offsetHeight * _list.offsetHeight; }
 };
+
+window.skyrimPlatform.sendMessage("front-loaded");
```

Comment:
if (window.skyrimPlaatform !== undefined)

Reply:
We have this https://github.com/skyrim-multiplayer/skymp/blob/8c589db687030db883cdcc8fb34d78e62a4bb1f9/skymp5-front/src/index.js#L15

Reply:
Added `if (window.skyrimPlatform?.sendMessage)`